src/recipe/dto/fetch-recipes-response.ts

```diff
@@ -1,16 +1,19 @@
-interface IFetchedRecipes {
-  id: number;
-  createdAt: Date;
-  title: string;
-  description: string;
-  ingredients: string;
-  preparation: string;
+class FetchedRecipe {
+  public id: number;
+  public createdAt: Date;
+  public title: string;
+  public description: string;
+  public ingredients: string;
+  public preparation: string;
 }
 
 export class FetchRecipesResponse {
-  constructor(public fetchedRecipes: IFetchedRecipes[]) {}
+  public fetchedRecipes: FetchedRecipe[];
+  constructor(fetchedRecipes: FetchedRecipe[]) {
+    this.fetchedRecipes = fetchedRecipes;
+  }
 
-  public static from(fetchedRecipes: IFetchedRecipes[]): FetchRecipesResponse {
+  public static from(fetchedRecipes: FetchedRecipe[]): FetchRecipesResponse {
```

Comment:
I think I already commented on this: don't use `FetchedRecipe` as input parameter - use the one that you are mapping from, this will probably be some prisma class